docs/specifications/tests/Consistency-TP/consistency05.md

```diff
@@ -1,49 +1,205 @@
-## CONSISTENCY05: Consistency between glue and authoritative data
+# CONSISTENCY05: Consistency between glue and authoritative data
 
-### Test case identifier
+## Test case identifier
 
-**CONSISTENCY05:** Consistency between glue and authoritative data
+**CONSISTENCY05**
 
-### Objective
+## Objective
 
 For name servers that have IP addresses listed as glue, the IP addresses must
-match the authoritative A and AAAA records for that host. This is an IANA
-requirement (https://www.iana.org/help/nameserver-requirements).
+match the authoritative A and AAAA records for that host. This is an IANA 
+[name server requirement].
+
+The objective of this test is to verify that the glue address records 
+in the delegation are consistent with authoritative data.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Ordered description of steps to be taken to execute the test case
+1. Obtain the set of name server names from the NS records in the 
+   delegation of *Child Zone* using [Method2] and any glue IP addresses
+   from the same delegation using [Method4].
+
+   1. Extract the [in-bailiwick] name server names and create the set
+      "Delegation Strict Glue", where each name server name 
+      is matched with its IP address or addresses, if available. The 
+      set may be empty.
```

Comment:
From the point of view of both programming and mathematics, the general rule is that sets may be empty unless otherwise stated. I believe a general technical audience would agree.

In any case this document is inconsistent with regards to notes on potential emptiness. It is noted for this set and for "Delegation Extended Glue", but not for "IB NS Name Set".